.github/workflows/check_extensions.yml

```
@@ -0,0 +1,39 @@
+name: Update extension list
+
+on:
+  workflow_dispatch: # Allow running on-demand
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '6 6 * * *' # every sunday
+
```

```suggestion
  pull_request:
    paths:
      - '.github/workflows/check_extenstions.yml'

```
Maybe also have it run when the workflow is edited in PR? might make maintenance easier?

---

Nice!